x-pack/test/functional/apps/maps/feature_controls/maps_security.ts

```diff
@@ -16,15 +16,16 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const queryBar = getService('queryBar');
   const savedQueryManagementComponent = getService('savedQueryManagementComponent');
 
-  // FLAKY: https://github.com/elastic/kibana/issues/38414
-  describe.skip('security feature controls', () => {
+  describe('maps security feature controls', () => {
     before(async () => {
       await esArchiver.loadIfNeeded('maps/data');
       await esArchiver.load('maps/kibana');
     });
 
     after(async () => {
       await esArchiver.unload('maps/kibana');
```

Comment:
Because this test removes data, the next test in the test sequence starts with an empty state.